src/js/common/actions/index.js

```diff
@@ -0,0 +1,7 @@
+export const LOG_OUT = 'LOG_OUT';
```

Comment:
The reason I'd moved it was because it seemed odd to include an action from one react component into another. But yes, it was sort of lonely there (which is probably why it was moved out of common and into login in the first place). In any case, I've moved it back into the login code and willing to live with importing an action from the login code into the user-profile code.